vite.config.js

```diff
@@ -14,7 +16,7 @@ if (
   )
 }
 
-const root = new URL('.', import.meta.url).pathname
+const root = dirname(fileURLToPath(import.meta.url))
 export default defineConfig({
   root,
```

Comment:
FWIW, this [can be a relative path too](https://github.com/vitejs/vite/blob/2dfc0152be6ed531fdebe7dc76314ac9f80b7b1d/packages/vite/src/node/config.ts#L72-L76):

```suggestion
  root: './',
```